inc/functions/profile-global-functions.php

```diff
@@ -152,5 +152,37 @@ function guard_functions_exist($arr){
     return $flag;
 }
 
-
+function profile_breadcrumbs() {
+    $permalink = get_permalink();
+    if ( $permalink !== network_site_url() ) {
+        $link = str_replace( network_site_url(), '', $permalink );
+    } else {
+        $link = null;
+    }
+    $link = rtrim($link, '/');
+    $link = ltrim($link, '/');
+    $link_parts = explode('/', $link);
+    $last = end($link_parts);
+    $url = '';
+    echo '<div class="breadcrumbs">';
+    if (is_home() || is_front_page()) {
+        echo '<span><a href="' . network_site_url() . '/' . '">Home</a></span>';
+    } else {
+        echo '<span><a href="' . network_site_url() . '/' . '">Home</a></span>';
+        echo ' <span class="sep">&gt;</span> ';
+        foreach ( $link_parts as $part ) {
+            $url .= $part . '/';
```

Comment:
Yes, we need that because the variable $url has to be initialized before using it inside the loop or otherwise will affect the string concatenation.